nl.js

```diff
@@ -0,0 +1,747 @@
+generic.username Gebruikersnaam Invoeren
+generic.password Wachtwoord Invoeren
+generic.register Registreren
+generic.login Inloggen
+generic.accuracy Nauwkeurigheid
+player.waiting wachten op spelers
+player.died Je stierf
+player.killed Gedood door
+player.killstreak {0} is op een {1} Kill Streak
+player.inactive Gekicked voor inactiviteit
+player.disconnect VERBINDING VERBROKEN
+timer.end Volgende ronde in {0}
+leaderboard.empty Lege Lobby
+popup.checkpoint Checkpoint!
+app.play KLIK OM TE SPELEN
+app.spectate KLIK OM TOE TE KIJKEN
+app.play.mobile TAP OM TE SPELEN
+app.spectate.mobile TAP OM TE SPECTATEN
+app.play.controller KLIK START
+app.spectate.controller START MET TOEKIJKEN
+purchase.error Fout opgetreden
+purchase.error.click klik hier</a> en probeer opnieuw!
+purchase.error.limit Je hebt mogelijk je dagelijkse transactielimiet overschreden
+settings.reset Reset
+settings.reset.confirm Weet je zeker dat je al je instellingen wilt resetten?
+settings.import Importeren
+settings.export Exporteren
+settings.none Geen instellingen gevonden
+settings.backup Backup Instellingen
+settings.controls.change Keybinds
+settings.controls.press Klik op een Willekeurige toets 
+settings.local.header Lokalisatie <span id="requiresRestart"><span style="color: #eb5656">*</span> opnieuw opstarten vereist</span>
+settings.local.region Standaard Regio
+settings.local.lang Taal
+settings.network.header Netwerk
+settings.network.sendRate Hoge Tickrate
+settings.network.netRerouting Auto Routing (Beïnvloedt Ping)
+settings.network.lagComp Lag Compensatie
+settings.popup.header Popups
+settings.popup.freeKRPop Toon Gratis KR Popup
+settings.experimental.header Experimenteel
+settings.experimental.aimFreezeFix Aim Freeze Fix
+settings.experimental.mouseFlickFix Muis Flick Fix
+settings.experimental.shaderRendering Instance Rendering
+settings.dev.header Developer
+settings.dev.forceUpdate Force Update Skins
+settings.dev.forceRegen Altijd Speler Regenereren
+settings.dev.extraBan Extra Ban Info
+settings.chat.header Chat
+settings.chat.chat Toon Chat Box
+settings.chat.chat.all Altijd
+settings.chat.chat.focus Terwijl Gefocused 
+settings.chat.messages Toon Speler Berichten
+settings.chat.unboxings Toon Unboxings
+settings.chat.chatOp Chat Opacity
+settings.chat.chatBGOp Chat BG Opacity
+settings.chat.chatHeight Chat Hoogte
+settings.chat.chatTimer Chat Timer (s)
+settings.chat.chatTextOutline Chat Text Outline
+settings.chat.chatFilter Laster Filter
+settings.quality.header Prestatie
+settings.quality.updRate Frame Limiet
+settings.quality.rendDst Render Afstand
+settings.quality.res Resolutie
+settings.quality.part Particles
+settings.quality.mapDet Map Details
+settings.quality.noTex Geen Texturen
+settings.quality.partdist Particle Afstand
+settings.quality.renderdist Render Afstand
+settings.quality.reflection Reflectie Kwaliteit
+settings.quality.antiAlias Antialiasing
+settings.quality.shadows Schaduw
+settings.quality.softShad Zachte Schaduw
```

Comment:
'Zachte Schaduw' should be changed to Zachte Schaduwen

Reply:
Done.